pkgs/leak_tracker_flutter_testing/pubspec.yaml

```diff
@@ -12,11 +12,12 @@ dependencies:
   flutter_test:
     sdk: flutter
   leak_tracker:  '>=9.0.4 <10.0.0'
-  leak_tracker_testing: '>=1.0.0 <2.0.0'
+  leak_tracker_testing: '>=1.0.4 <2.0.0'
```

Comment:
Ubernit: can this just be `^1.0.4`? Typically we avoid explicit ranges that only span a single major revision.